src/pr.ts

```diff
@@ -23,14 +23,12 @@ const localize: nls.LocalizeFunc = nls.loadMessageBundle();
  */
 export namespace fs {
 
-    export function exists(fspath: string): Promise<boolean> {
-        return new Promise<boolean>((resolve, _reject) => {
-            fs_.exists(fspath, res => resolve(res));
-        });
+    export async function exists(filePath: string): Promise<boolean> {
+        return (await tryStat(filePath))?.isFile() || false;
```

Comment:
Should `exists()` really check for `isFile()`? And if so, shouldn't `existsSync()` do the same?

Reply:
No, it shouldn't. That's why the CI failed.  I just hadn't had the time to come back to this and was planning to today.  I wrote the code with a file in mind and that's where the mistake came from.  Thanks for the comment.